src/flexmeasures_client/client.py

```diff
@@ -329,3 +329,32 @@ async def trigger_and_get_schedule(
         )
 
         return schedule
+
+    async def get_sensor_data(
+        self,
+        sensor_id: int,
+        start: str | datetime,
+        duration: str | timedelta,
+        unit: str,
+        entity_address: str,
```

Comment:
Could we derive this instead?

Reply:
Can you tell me how to derive the entity address or where I can find how it's created?

Reply:
https://flexmeasures.readthedocs.io/en/latest/api/notation.html#sensors-and-entity-addresses

But let's keep it simple, because we are phasing out these entity addresses. In fact, most of what is in there is already ignored by FlexMeasures.

`entity_address = f"ea1.2021-01.io.flexmeasures:fm1.{sensor_id}"` should do the trick.